theme.py

```diff
@@ -390,16 +410,16 @@ def apply(self, root):
                 window.setAppearance_(appearance)
 
         elif sys.platform == 'win32':
-            GWL_STYLE = -16
-            WS_MAXIMIZEBOX = 0x00010000
+            GWL_STYLE = -16  # noqa: N806 # ctypes
+            WS_MAXIMIZEBOX = 0x00010000  # noqa: N806 # ctypes
             # tk8.5.9/win/tkWinWm.c:342
-            GWL_EXSTYLE = -20
-            WS_EX_APPWINDOW = 0x00040000
-            WS_EX_LAYERED = 0x00080000
-            GetWindowLongW = ctypes.windll.user32.GetWindowLongW
-            SetWindowLongW = ctypes.windll.user32.SetWindowLongW
+            GWL_EXSTYLE = -20  # noqa: N806 # ctypes
+            WS_EX_APPWINDOW = 0x00040000  # noqa: N806 # ctypes
+            WS_EX_LAYERED = 0x00080000  # noqa: N806 # ctypes
+            GetWindowLongW = ctypes.windll.user32.GetWindowLongW  # noqa: N806 # ctypes
+            SetWindowLongW = ctypes.windll.user32.SetWindowLongW  # noqa: N806 # ctypes
 
-            root.overrideredirect(theme and 1 or 0)
+            root.overrideredirect(theme and True or False)
```

Comment:
```suggestion
            root.overrideredirect(bool(theme))
```

Reply:
Possibly.  Mostly I want to entirely change the "theme is an int with magic meanings for the values" at some point.

Reply:
I'm definitely going to follow up merging this with changing all the `0`, `1`, `2` for theme selection to using defined constants, and *not* doing any clever "oh, default is `0`, so let's treat this like a boolean" bullshit.